arrow-libs/core/arrow-raise-contextual/src/commonMain/kotlin/arrow/core/raise/contextual/Raise.kt

```diff
@@ -0,0 +1,125 @@
+@file:OptIn(ExperimentalTypeInference::class, ExperimentalContracts::class)
+
+package arrow.core.raise.contextual
+
+import arrow.core.Either
+import arrow.core.NonEmptyList
+import arrow.core.NonEmptySet
+import arrow.core.Option
+import arrow.core.Some
+import arrow.core.raise.*
+import kotlin.contracts.ExperimentalContracts
+import kotlin.contracts.contract
+import kotlin.experimental.ExperimentalTypeInference
+import kotlin.jvm.JvmName
+import arrow.core.raise.Raise as CoreRaise
+import arrow.core.raise.SingletonRaise as CoreSingletonRaise
+import arrow.core.raise.ResultRaise as CoreResultRaise
+
+public typealias Raise<Error> = CoreRaise<Error>
+public typealias SingletonRaise<Error> = CoreSingletonRaise<Error>
+public typealias ResultRaise = CoreResultRaise
```

Comment:
Is the purpose of these to allow consumers to use only `import arrow.core.raise.contextual.*` and avoid clashes with `arrow.core.raise.*`?